CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `5.8.0`
```

Comment:
Please change this header to `## Recent Changes` and our CI will bump the version automatically when the PR is merged.

Reply:
Header changed with `## Recent Changes`